exotations/solvers/exotica_levenberg_marquardt_solver/include/exotica_levenberg_marquardt_solver/levenberg_marquardt_solver.h

```diff
@@ -1,9 +1,39 @@
+//
```

Comment:
I think the LM solver is MIT licensed, i.e., this is incorrect:
https://github.com/ipab-slmc/exotica/blob/bb96e3a8338ef64fe7243c215457ac036f9d57ec/exotations/solvers/exotica_levenberg_marquardt_solver/package.xml#L9